font.css

```diff
@@ -0,0 +1,40 @@
+@font-face {
+font-family: 'montserratbold';
+src: url('fonts/montserrat-bold.woff2') format('woff2'),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/tesla394/pullRequest?prid=5642156)